fiftyone/utils/eval/detection.py

```
@@ -30,6 +31,45 @@
+    eval_key,
+    save=False,
+    processing_frames=False,
+    tp_field=None,
+    fp_field=None,
+    fn_field=None,
+) -> List[Any]:
+    matches = []
+    if processing_frames:
+        docs = sample.frames.values()
+    else:
+        docs = [sample]
```

Ruff (0.8.2)

Use ternary operator `docs = sample.frames.values() if processing_frames else [sample]` instead of `if`-`else`-block

Replace `if`-`else`-block with `docs = sample.frames.values() if processing_frames else [sample]`

(SIM108)